blog/2025-06-23-june.md

```diff
@@ -2,7 +2,7 @@
 slug: 2025-06-24-june
 title: "Masumi Interview"
 authors: [denicio]
-tags: [ai]
+tags: [AI]
```

Comment:
While we did not state this anywhere, tags are typically all lower case, because people are otherwise not consistent in their capitalization and then the build breaks. (same for tags in builder tools and showcase)